101-automation/azuredeploy.json

```diff
@@ -0,0 +1,207 @@
+            "description": "Automation account name"
+        }
+    },
+    "automationAccountLocation": {
+        "type": "string",
+        "metadata": {
+            "description": "Specify the location in which to create the Automation account."
+        }
+    },
+	"sampleGraphicalRunbookName": {
+            "type": "String",
+			"defaultValue": "AzureAutomationTutorial"
+        },
+        "sampleGraphicalRunbookDescription": {
+            "type": "String",
+			"defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "sampleGraphicalRunbookContentUri": {
+            "type": "String",
+			"defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorial.graphrunbook"
+        },
+        "samplePowerShellRunbookName": {
+            "type": "String",
+			"defaultValue": "AzureAutomationTutorialScript"
+        },
+        "samplePowerShellRunbookDescription": {
+            "type": "String",
+			"defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "samplePowerShellRunbookContentUri": {
+            "type": "String",
+			"defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorial.ps1"
+        },
+        "samplePython2RunbookName": {
+            "type": "String",
+			"defaultValue": "AzureAutomationTutorialPython2"
+        },
+        "samplePython2RunbookDescription": {
+            "type": "String",
+			"defaultValue": " An example runbook that gets all the Resource Manager resources by using the Run As account (service principal)."
+        },
+        "samplePython2RunbookContentUri": {
+            "type": "String",
+			"defaultValue": "https://eus2oaasibizamarketprod1.blob.core.windows.net/marketplace-runbooks/AzureAutomationTutorialPython2.py"
+		}
+},
+"resources": [
+    {
+    "type": "Microsoft.OperationalInsights/workspaces",
+        "name": "[parameters('workspaceName')]",
+        "apiVersion": "2020-03-01-preview",
+        "location": "[parameters('location')]",
+        "properties": {
+            "sku": {
+                "name": "[parameters('sku')]",
+            },
+            "retentionInDays": "[parameters('dataRetention')]",
+            "features": {
+                "searchVersion": 1,
+                "legacy": 0,
+                "enableLogAccessUsingOnlyResourcePermissions": true
+            }
+        },
+	"resources": [
+    {
+        "type": "Microsoft.Automation/automationAccounts",
+        "apiVersion": "2018-06-30",
+        "name": "[parameters('automationAccountName')]",
+        "location": "[parameters('automationAccountLocation')]",
+        "dependsOn": [
+		     "[concat('Microsoft.OperationalInsights/workspaces/', parameters('workspaceName'))]",
+		],
+        "tags": {},
```

Comment:
remove all empty/null properties throughout the template (see CI results)

Reply:
Done